docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -119,6 +119,42 @@ This property is optional; the default is ``1m``.
 Enable or disable using the address published by Elasticsearch to connect for
 queries.
 
+``elasticsearch.union-schema-indices-for-alias``
```

Comment:
Similar to my comments on `elasticsearch.datatype-for-alias-schema-mismatch`, I think exposing this property adds more confusion than helps. I'd rather have a default functionality of the raw json that is well documented and has all the json properties to parse it. 

Perhaps in a future fix, we can add the option to put an "expected type" mapping.

Reply:
Reason for exposing this property is that we currently have a behaviour of taking only first index mapping for an alias table. Dont wanted to deviate from the existing behaviour without giving users the option to migrate. Hence I added this property. I upvote to keep this behaviour and maybe after couple of releases get rid of it.

Reply:
Ah yes, that makes sense. I think I misinterpreted the default behavior. We should verify if this property is turned off that this disables all new features, including creation of cache, etc... Also we should make sure the other settings depend on this one.

Reply:
Thats already taken care. All the further settings depend on `union-schema-indices-for-alias` as explained in the [description](https://github.com/trinodb/trino/pull/8202#issue-662460462) of this PR.